### PR TITLE
Making sure readwritesqlite is in good working order

### PR DESCRIPTION
**Pull request by dunkenwg**

### CODE REVIEW

**Comment by dunkenwg** [comment]
I nested several `expect_warnings()` to remove the warnings that were popping up in "test-write.R"

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-    branches:
-      - main
-      - master
-      - dev
-  schedule:
-    - cron:  '0 8 * * *'
+    branches: [main, master]
 
-name: R-CMD-check
+name: R-CMD-check.yaml
```
Comment (joethorley) [inline .github/workflows/R-CMD-check.yaml:13]:
```suggestion
name: R-CMD-check
```

I'd really like to drop the .yaml from the name. @aylapear what do you think?

**Review by joethorley — CHANGES_REQUESTED** [review]
This is really great.

A few thoughts:

- Its important to do styler changes separately from functional changes as it hard to parse the code visually to tell which is what.
- I've got one suggested change to the R-CMD-check.yaml but would like to get Ayla's thought on it
- My final question is what were the warnings that were suppressed in the tests?

Thanks

File: .github/workflows/R-CMD-check.yaml (hunk 1 above)
Comment (aylapear) [reply .github/workflows/R-CMD-check.yaml:13]: Yeah, that is a great catch Joe. We should file an issue with them if it hasn't already been done.

**Comment by dunkenwg** [comment]
Several code chunks now have nested `expect_warnings()`. I did this because we were originally looking for multiple warnings in the same function call separately.

For example, the way the code was written:
```
expect_warning(
    rws_write(env, strict = FALSE, conn = conn),
    "^The following data frame in 'x' is unrecognised: 'local2'; but exists = TRUE[.]$"
  )
expect_warning(
    rws_write(env, strict = FALSE, conn = conn),
    "^The following column in data 'local' is unrecognised: 'z'[.]$"
  )
```
outputs `Warning message: The following column in data 'local' is unrecognised: 'z'. ` for the first `expect_warning()` and `Warning message: The following data frame in 'x' is unrecognised: 'local2'; but exists = TRUE.` for the second `expect_warning()`.

The way the code is written now:
```
expect_warning(
    {
      expect_warning(
        rws_write(env, strict = FALSE, conn = conn),
        "The following data frame in 'x' is unrecognised: 'local2'; but exists = TRUE"
      )
    },
    "The following column in data 'local' is unrecognised: 'z'"
  )
```
captures both warnings and doesn't give any extra warning messages.

File: .github/workflows/R-CMD-check.yaml (hunk 1 above)
Comment (joethorley) [reply .github/workflows/R-CMD-check.yaml]: OK - great -I've filed an issue in our internal infrastructure repository for this.

**Review by joethorley — APPROVED** [review]
Thanks!

**Review by aylapear — APPROVED** [review]
Looks good to me
